pkgs/development/compilers/zig/0.10.nix

```
@@ -77,6 +77,10 @@ stdenv.mkDerivation rec {
     $out/bin/zig test --cache-dir "$TMPDIR" -I $src/test $src/test/behavior.zig
   '';
 
+  passthru.buildZigPackage = import ../../../build-support/zig/build-zig-package/default.nix {
+    zig = finalAttrs.finalPackage;
```

you just need to pass in all the arguments manually

```suggestion
    inherit lib stdenv;
    zig = finalAttrs.finalPackage;
```

---

Oh, thanks!